### ISSUE TITLE
Third and fourth struct elements of an array initializer are put on the same line

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#49]
Somehow only the fourth array element gets placed on the same line as the others. Also if you extend the array to lets say 10 elements, only the fourth one has this peculiar behavior.

```
// in 
anArray : ARRAY[1..5] OF Structure := [
	(Aab:=744.00, Aac:=496.00, Aad:=225.00, Aaf:=0, Aag:=1),
	(Aab:=744.00, Aac:=496.00, Aad:=225.00, Aaf:=0, Aag:=1),
	(Aab:=759.00, Aac:=509.00, Aad:=222.00, Aaf:=0, Aag:=1), 
	(Aab:=753.00, Aac:=490.00, Aad:=226.00, Aaf:=0, Aag:=1),
	(Aab:=742.00, Aac:=493.00, Aad:=223.00, Aaf:=0, Aag:=1)
]

// out
anArray : ARRAY[1..5] OF Structure := [
	(Aab:=744.00, Aac:=496.00, Aad:=225.00, Aaf:=0, Aag:=1),
	(Aab:=744.00, Aac:=496.00, Aad:=225.00, Aaf:=0, Aag:=1),
	(Aab:=759.00, Aac:=509.00, Aad:=222.00, Aaf:=0, Aag:=1), (Aab:=753.00, Aac:=490.00, Aad:=226.00, Aaf:=0, Aag:=1),
	(Aab:=742.00, Aac:=493.00, Aad:=223.00, Aaf:=0, Aag:=1)
];
```

If I change "Break if line length overflow" to "Break if line length overflow or multiline" it works "as expected". Is this intended behavior, that only the fourth element is placed on the same line as the previous element?

**STweep version info**
STweep for TwinCAT 3.6.5

**IDE version info**
Microsoft Visual Studio Community 2017 Version 15.9.35

**STWeep settings**
[roald.txt](https://github.com/Barteling/STweep/files/7468183/roald.txt)

**Comment by Barteling** [comment]
Thanks for letting me know.
I can repeat it, and this is definitely not intended.

I'll dig in to this.


Note to self:
With the attached set of setting I even get an formatting exception with the above example.

[Throws.STweep.txt](https://github.com/Barteling/STweep/files/7475947/Throws.STweep.txt)

**Comment by Barteling** [comment]
I fixed this is Stweep for TwinCAT 4.0.0-beta.
https://www.stweep.com/beta-releases/

The update for the Codesys version will follow soon.
